### ISSUE TITLE
Show image description character limit

### ISSUE THREAD

**Issue by ghost** [nolanlawson/pinafore#259]
See tootsuite/mastodon#7339 - The character limit of (seemingly?) 420 for image descriptions is not displayed anywhere. Main difference between this and the issue on the mastodon github is on pinafore, you are also not stopped from typing at the character limit. That's fine if the character limit gets displayed, but otherwise is not great.

**Comment by nolanlawson** [comment]
You're right; we need to prevent people posting descriptions that are too long (which I assume will just fail with a server error).

**Comment by nolanlawson** [comment]
This is somewhat fixed now that there's a media edit dialog. Inside of the media edit dialog, it shows you how many characters you have left relative to 420.

I could just add `maxlength=420` to the main `textarea` for media descriptions as well, but I'm not sure if that will play well with emoji character lengths, so it needs some investigation.

**Comment by cincodenada** [comment]
I'm not sure if I should comment here or open a new issue, but this setting varies by instance, and Pinafore doesn't seem to query the instance. For example, my instance (cybre.space) allows 1500 characters but Pinafore limits me to 420.
